platform/android/display_server_android.cpp

```diff
@@ -484,16 +482,16 @@ DisplayServerAndroid::~DisplayServerAndroid() {
 void DisplayServerAndroid::process_joy_event(DisplayServerAndroid::JoypadEvent p_event) {
 	switch (p_event.type) {
 		case JOY_EVENT_BUTTON:
-			Input::get_singleton()->joy_button(p_event.device, p_event.index, p_event.pressed);
+			Input::get_singleton()->joy_button(p_event.device, (JoyButton)p_event.index, p_event.pressed);
 			break;
 		case JOY_EVENT_AXIS:
 			Input::JoyAxisValue value;
 			value.min = -1;
 			value.value = p_event.value;
-			Input::get_singleton()->joy_axis(p_event.device, p_event.index, value);
+			Input::get_singleton()->joy_axis(p_event.device, (JoyAxis)p_event.index, value);
```

Comment:
`Input::JoyAxisValue`, `JoyAxis`
Maybe being explicit by writing `Input::` is desired in here? No idea if that's the case.

Reply:
I'm not sure which is desired, it's up to @akien-mga's personal preference I guess.

Reply:
I think it's fine as is. `JoyAxis`, `JoyButton`, etc. are extremely common enums used as global throughout the codebase. `JoyAxisValue` is a very specific struct that most contributors are not familiar with, it's good to be explicit.